docs-src/concepts/what-is-persistence.md

```diff
@@ -0,0 +1,46 @@
+---
+id: what-is-persistence
```

Comment:
This one I am more open to as a definable term. We persist very specific things.